GUI/SearchTaskWindow.xaml

```diff
@@ -815,6 +815,13 @@
                                                     </TextBlock>
                                                 </RadioButton.ToolTip>
                                             </RadioButton>
+                                            <RadioButton x:Name="massDiffAcceptPlusOrMinusThree" Content="+- 3 Missed Monoisotopic Peaks" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="500">
```

Comment:
this adds a new mass diff acceptor. plus or minus three daltons. it's good for top-down searches where the missed monos are in both directions